<ui version="4.0" >
 <author>Andrzej Zawadzki</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>29</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>7</hsizetype>
    <vsizetype>7</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>FANN Console</string>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="aboutAction" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="newAction" />
    <addaction name="openAction" />
    <addaction name="saveAction" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuHelp" />
  </widget>
  <action name="newAction" >
   <property name="text" >
    <string>New Network</string>
   </property>
  </action>
  <action name="openAction" >
   <property name="text" >
    <string>Open Network</string>
   </property>
  </action>
  <action name="saveAction" >
   <property name="text" >
    <string>Save Network</string>
   </property>
  </action>
  <action name="aboutAction" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
